docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -0,0 +1,254 @@
+title: Get started with Azure Blob storage using F#
+description: Store unstructured data in the cloud with Azure Blob storage.
+keywords: visual f#, f#, functional programming, .NET, .NET Core, Azure
+author: sylvanc
+manager: jbronsk
+ms.date: 09/20/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: c5b74a4f-dcd1-4849-930c-904b6c8a04e1
+---
+
+# Get started with Azure Blob storage using F# #
+
+Azure Blob storage is a service that stores unstructured data in the cloud as objects/blobs. Blob storage can store any type of text or binary data, such as a document, media file, or application installer. Blob storage is also referred to as object storage.
+
+### About this tutorial
+
+This article shows you how to perform common tasks using Blob storage. The samples are written using F# using the Azure Storage Client Library for .NET. The tasks covered include how to upload, list, download, and delete blobs.
+
+### Conceptual overview
+
+For a conceptual overview of blob storage, please see [the .NET guide for blob storage](https://azure.microsoft.com/en-us/documentation/articles/storage-dotnet-how-to-use-blobs/)
+
+### Create an Azure storage account
+
+To use this guide, you must first [create an Azure storage account](https://azure.microsoft.com/en-us/documentation/articles/storage-create-storage-account/).
+You'll also need your storage access key for this account.
+
+## Create an F# Script and Start F# Interactive
+
+The samples in this article can be used in either an F# application or an F# script. To create an F# script, create a file with the `.fsx` extension, for example `blobs.fsx`, in your F# development environment.
+
+Next, use a [package manager](package-management.md) such as Paket or NuGet to install the `WindowsAzure.Storage` package and reference `WindowsAzure.Storage.dll` in your script using a `#r` directive.
+
+### Add namespace declarations
+
+Add the following `open` statements to the top of the `blobs.fsx` file:
+
+[!code-fsharp[BlobStorage](../../../samples/snippets/fsharp/azure/blob-storage.fsx#L1-L5)]
+
+### Get your connection string
+
+You'll need an Azure Storage connection string for this tutorial. For more information about connection strings, see [Configure Storage Connection Strings](https://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/).
+
+For the tutorial, you'll enter your connection string in your script, like this:
+
+[!code-fsharp[BlobStorage](../../../samples/snippets/fsharp/azure/blob-storage.fsx#L11-L11)]
+
+However, this is a **bad idea** for real projects. Your storage account key is similar to the root password for your storage account. Always be careful to protect your storage account key. Avoid distributing it to other users, hard-coding it, or saving it in a plain-text file that is accessible to others. You can regenerate your key using the Azure Portal if you believe it may have been compromised.
```

Comment:
Instead of saying this is a bad idea, wouldn't be better to say it is not recommended?